examples/src/test/java/org/neo4j/docs/driver/ExamplesIT.java

```diff
@@ -488,4 +491,106 @@ void testHostnameVerificationExample()
             assertTrue( example.canConnect() );
         }
     }
+
+    @Test
+    @EnabledOnNeo4jWith( BOLT_V4 )
+    void testShouldRunRxAutocommitTransactionExample() throws Exception
+    {
+        try ( RxAutocommitTransactionExample example = new RxAutocommitTransactionExample( uri, USER, PASSWORD ) )
+        {
+            // create some 'Product' nodes
+            try ( Session session = neo4j.driver().session() )
+            {
+                session.run(
+                        "UNWIND ['Tesseract', 'Orb', 'Eye of Agamotto'] AS item " +
+                                "CREATE (:Product {id: 0, title: item})" );
+            }
+
+            // read all 'Product' nodes
+            List<String> titles = await( example.readProductTitlesReactor() );
```

Comment:
I'd use the Project reactors `StepVerifier` for Reactors test and probably similar things if there any for RxJava2.